loopy/kernel/array.py

```diff
@@ -691,7 +691,9 @@ def __init__(self, name, dtype=None, shape=None, dim_tags=None, offset=0,
             using atomic-capable data types.
         :arg offset: (See :attr:`offset`)
         :arg alignment: memory alignment in bytes
-
+        :arg tags: A metadata tag or list of metadata tags intended for
+            consumption by an application. These could be strings or instances
+            of :class:`pytools.tag.Tag` for example.
```

Comment:
Disagree with this. All tags should derive from `pytools.tag.Tag`.

Reply:
Changed to wording to only mention `pytools.tag.Tag`.